### ISSUE TITLE
[Feature Request]: make ose-oauth-proxy container version or sha256 digest reference consistent across all components odh-dashboard

### ISSUE THREAD

**Issue by shalberd** [opendatahub-io/odh-dashboard#1040]
### Feature description

@LaVLaS @andrewballantyne @strangiato 

https://github.com/opendatahub-io/odh-manifests/search?q=ose-oauth-proxy

In the ODH-Dashboard, there currently is an oauth sidecar with version 4.8

`registry.redhat.io/openshift4/ose-oauth-proxy:v4.8`

This is just for the route to ODH dashboard and authentication, not for jupyter notebooks / kubeflow notebooks in 

odh-notebook-controller

`registry.redhat.io/openshift4/ose-oauth-proxy:v4.10`

odh-prometheus

`registry.redhat.io/openshift4/ose-oauth-proxy@sha256:4bef31eb993feb6f1096b51b4876c65a6fb1f4401fee97fa4f4542b6b7c9bc46`

data science pipelines ui

egistry.redhat.io/openshift4/ose-oauth-proxy:v4.8

It is not good for reproducability of there are different oauth-proxy images used.

You can look up versions and tags/sha256 digest values here

https://catalog.redhat.com/software/containers/search?q=ose-oauth-proxy

select architecture AMD64

search for a tag, e.g. v4.10 (or 4.12?) what is your Openshift target currently?

![Bildschirmfoto 2023-03-24 um 16 58 10](https://user-images.githubusercontent.com/21118431/227579846-51b488cc-984a-42dc-8987-d0890046fd24.png)

get the "Manifest List Digest" value, that is architecture-neutral. @Jooho I assume in the image digest value, the manifest list images by digest value must be used, but I could be wrong.

To summarize:
if using tags, think about what tag that is congruent with primar openshift version

if using digests, first look up the tag and then fill in the digest value (Manifest List Digest)

containerStatuses of a pod resolves like this, so I think using the full string and digest from "Manifest List Digest" is the way to go

```
imageID: >-
        registry.redhat.io/openshift4/ose-oauth-proxy@sha256:792f9e2db2d25c63ceb2169e38209622062f5c431522cfe85976ca23b1dc8c57
      image: 'registry.redhat.io/openshift4/ose-oauth-proxy:v4.10'

```

**Comment by shalberd** [comment]
moving to odh-manifests
